RELEASE_NOTES.md

```diff
@@ -0,0 +1,41 @@
+# Release Notes v2.0.0 (Dec 16 2024)
+
+This the first release for Greengrass nucleus lite (GGLite) product line. GGLite
+v2 aims to be compatable with AWS's GreenGrass v2(GGV2) product line, however
+currently only subset of the features are supported with this release.
+
+## Installing from sources
+
+To install GGLite from source, please follow the guide from
+[SETUP.md](./docs/SETUP.md) and [TES.md](./docs/TES.md), once the setup
+environment is complete, please start refering to
```

Comment:
```suggestion
 is complete, please start refering to
```

Reply:
I want to keep the setup environment as this is in relation with setting up a environment or the follow up where a developer can setup a nix environment